src/dfx/src/lib/installers/assets/mod.rs

```diff
@@ -37,5 +44,15 @@ pub async fn post_install_store_assets(
             )
         })?;
 
+    canister
+        .update_("enable_redirect")
+        .with_arg(EnableRedirectArguments {
+            enable: assets_canister_info.get_redirect(),
+        })
+        .build()
+        .call_and_wait(waiter_with_timeout(expiry_duration()))
+        .await
+        .context("Failed call to enable_redirect for asset canister.")?;
+
```

Comment:
if this feature should get implemented as part of `.ic-assets.json` file, then we might as well do it as part of https://github.com/dfinity/sdk/pull/2414 and provide users with default configuration like:
```json5
[
    {
        "match":"**/*", // match all files 
        "redirect":{
        "from":{
                "path":"(.*)\\.html" // if path contains ".html"
        },
        "to":{
                "path":"${1}" // strip ".html"
        },
        "response_code":301
    }
]
```

The issue with solution in #2414 is that the request send to the certified endpoint (ic0.app) in order to get properly redirected would need to go through consensus (slooow), where this solution (afaiu) deals with redirects internally (actually it's more of an aliasing then redirects I guess?). Would be nice to incorporate both solutions into something consistent, e.g.:
```json5
[
    {
        "match":"**/*", 
        "redirect":{
        "from":{
                "path":"(.*)\\.html" 
        },
        "to":{
                "path":"${1}"
        },
        "kind": "internal"  
    },
    {
        "match":"**/*", 
        "redirect":{
        "from":{
                "path":"(.*)\\.html" 
        },
        "to":{
                "path":"${1}" 
         },
        "kind": "external",
        "response_code": 301 // can only be used when kind==external  
    },
]
```

I have zero clue what these changes entail in terms of certifying the request - if someone already knows this please explain, otherwise ask me in 3 weeks when I'll learn more about this